packages/next/src/client/dev/dev-build-watcher.ts

```diff
@@ -6,18 +6,19 @@ export default function initializeBuildWatcher(
   toggleCallback: any,
   position = 'bottom-right'
 ) {
-  const shadowHost = document.createElement('div')
+  const type = 'div' as string
+  const shadowHost = document.createElement(type)
   const [verticalProperty, horizontalProperty] = position.split('-')
   shadowHost.id = '__next-build-watcher'
   // Make sure container is fixed and on a high zIndex so it shows
   shadowHost.style.position = 'fixed'
   // Ensure container's position to be top or bottom (default)
-  ;(shadowHost as any).style[verticalProperty] = '10px'
+  shadowHost.style[verticalProperty as any] = '10px'
   // Ensure container's position to be left or right (default)
-  ;(shadowHost as any).style[horizontalProperty] = '20px'
-  ;(shadowHost as any).style.width = 0
-  ;(shadowHost as any).style.height = 0
-  ;(shadowHost as any).style.zIndex = 99999
+  shadowHost.style[horizontalProperty as any] = '20px'
```

Comment:
Oh you might be right about declaring the type to be literal instead of just `string`. 

Let me try that <picture data-single-emoji=":inspector-gadget:" title=":inspector-gadget:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/inspector-gadget/be3a8dd5b4bde146.png" alt=":inspector-gadget:" align="absmiddle"></picture>

Reply:
@huozhi This should do the trick: https://github.com/vercel/next.js/pull/54124